command/operator_debug.go

```diff
@@ -343,6 +348,7 @@ func (c *OperatorDebugCommand) Run(args []string) int {
 	flags.Int64Var(&eventIndex, "event-index", 0, "")
 	flags.StringVar(&eventTopic, "event-topic", "none", "")
 	flags.StringVar(&interval, "interval", "30s", "")
+	flags.StringVar(&pprofInterval, "pprofInterval", "30s", "")
```

Comment:
This should probably be `pprof-interval` so that it matches the other multi-word args. It might be nice to move this down next to the `pprof-duration` arg as well, because the behavior is paired together.